app/routes/vaccinate.js

```diff
@@ -53,6 +53,33 @@ module.exports = router => {
     res.redirect('/vaccinate/delivery-team')
   })
 
+  router.get('/vaccinate/delivery-team', (req, res) => {
+    let errors = []
+
+    if (req.query.showErrors === "yes") {
+      if (!req.session.data.deliveryTeam) {
+        errors.push({
+          text: "Select team",
```

Comment:
Suggestion from @Anna-Sutton:

```suggestion
          text: "Select a team",
```